x-pack/plugins/lens/public/editor_frame_plugin/plugin.tsx

```diff
@@ -33,10 +33,10 @@ class EditorFramePlugin {
         // on it's own because we are loosing type information here.
         // So it's basically explicitly saying "I'm dropping the information about type T here
         // because this information isn't useful to me." but without using any which can leak
-        this.datasources[name] = datasource as Datasource<unknown>;
+        this.datasources[name] = datasource as Datasource<unknown, unknown>;
       },
       registerVisualization: (name, visualization) => {
-        this.visualizations[name] = visualization as Visualization<unknown>;
+        this.visualizations[name] = visualization as Visualization<unknown, unknown>;
```

Comment:
I think these are both fine, but I also wonder if we should throw an error or log or something if the name is already taken. Otherwise, we could have accidental overrides. Do we want to override OSS registrations from x-pack? Should overrides be explicit?